Warn about using XIM as input method with wxGTK

vadz opened the pull request:
This results in various problems, including both the problems with
handling keyboard input (see #16840) and, even less expectedly,
graphical issues with flickering (see #18462).

Fixing or working around these issues doesn't seem to be easily
possible, but we can at least warn people about the problem.

vadz committed:
Warn about using XIM as input method with wxGTK

This results in various problems, including both the problems with
handling keyboard input (see #16840) and, even less expectedly,
graphical issues with flickering (see #18462).

Fixing or working around these issues doesn't seem to be easily
possible, but we can at least warn people about the problem.

PBfordev commented:
Maybe this deserves a FAQ entry in the [GTK section](http://wxwidgets.org/docs/faq/gtk/). But people probably read FAQs less then ever. BTW, I noticed you changed "GTK+" to "GTK" in the recent GTK docs change but FAQ still has "GTK+".

Speaking of FAQs, I wonder how up-to-date those Motif and X11 sections are.

vadz commented:
> Maybe this deserves a FAQ entry in the [GTK section](http://wxwidgets.org/docs/faq/gtk/). 

Good point, I've added it.

> But people probably read FAQs less then ever. BTW, I noticed you changed "GTK+" to "GTK" in the recent GTK docs change but FAQ still has "GTK+".

Also fixed this too.
 
> Speaking of FAQs, I wonder how up-to-date those Motif and X11 sections are.

Probably completely outdated, but then I also suspect not many people are looking at them anyhow...

PBfordev commented:
> Probably completely outdated, but then I also suspect not many people are looking at them anyhow...

I believe that general and MSW-specific parts are also ready for update. I would do that but it would probably fall on you to review and commit. Should I still do that?

vadz commented:
Of course, any help would be welcome, TIA!
